docs/source/faq.md

```diff
@@ -502,102 +428,41 @@ Back to the top
-Yes, if the software you need is not listed on our Installed Software page, you may submit a support request to have it installed by Research Computing staff. Please observe the following guidelines:
-
-Provide a link to the website from which to download the software If there are multiple versions, be specific about the version you want Let us know if you require any options that are not a standard part of the application If the effort required to install the software is 4 hours or less, the request is placed in the work queue to be installed once an RC staff member is available to perform the work, usually within a few business days.
-
-If initial evaluation of the request reveals that the effort is significantly greater than 4 hours, we will contact you to discuss how the work can be performed. It may be necessary to hire Research Computing staff as a consulting service to complete large and complex projects.
-
-You may also install applications yourself in your home directory.
-
-Please only ask us to install applications that you know will meet your needs and that you intend to use extensively. We do not have the resources to build applications for testing and evaluation purposes.
-::::
+You can submit a [Software Installation Request](https://bit.ly/NURC-Software) ticket to the Research Computing team. 
 
-::::{dropdown} Why do I get the 'command not found' error message?
-The Linux command interpreter (shell) maintains a list of directories in which to look for commands that are entered on the command line. This list is maintained in the PATH environment variable. If the full path to the command is not specified, the shell will search the list of directories in the PATH environment variable and if a match is not found, you will get the “command not found” message. A similar mechanism exists for dynamically linked libraries using the LD_LIBRARY_PATH environment variable.
-
-To ease the burden of setting and resetting environment variables for different applications, we have installed a “modules” system. Each application has an associated module which, when loaded, will set or reset whatever environment variables are required to run that application – including the PATH and LD_LIBRARY_PATH variables.
-
-The easiest way to avoid “command not found” messages is to ensure that you have loaded the module for your application. See Modules for more information.
+You can also try and install your program locally with package mangers or compiling from source by following our documentation pages.
+:::{seealso}
+{ref}`package-managers` and {ref}`from-source`
+:::
 ::::
 
 ::::{dropdown} Can I install my software on the HPC cluster?
-
-Yes, you can. Please follow the guidelines in the 'Installing Your Own Software' section. If you encounter any issues, contact support.
-
+Yes, you can. Please follow the guidelines in the {ref}`package-managers` and {ref}`from-source` sections. If you encounter any issues, contact Research Computing.
```

Comment:
Might be a good idea to add an email link to Research Computing, just to make it easier, but it's up to you!